mobile/src/pages/FeedPage.tsx

```diff
@@ -0,0 +1,54 @@
+import { useState } from "react";
+import { View } from "react-native";
+import { FlatList } from "react-native-gesture-handler";
+import { SafeAreaView } from "react-native-safe-area-context";
+import BottomNavBar from "../components/BottomNavBar";
+import Card from "../components/Card";
+import TopNavBar from "../components/TopNavBar";
+import useInfiniteQueryPosts from "../hooks/useInfiniteQueryPosts";
+import useDebounce from "../hooks/useDebounce";
+
+export default function FeedPage({ navigation }) {
+  const [search, setSearch] = useState("");
+
+  const debouncedSearch = useDebounce(search, 500);
+  const blob = useInfiniteQueryPosts({
+    searchValue: debouncedSearch,
+    limit: 8,
+    recentOnly: false,
+    key: "feedPage",
```

Comment:
We should pass in a better key to represent the content and avoid collisions with other keys (something like `feed_posts`)